shell/platform/windows/dpi_utils.cc

```diff
@@ -102,6 +99,10 @@ UINT Win32DpiHelper::GetDpiForWindow(HWND hwnd) {
 
 UINT Win32DpiHelper::GetDpiForMonitor(HMONITOR monitor) {
   if (dpi_for_monitor_supported_) {
+    if (monitor == nullptr) {
+      const POINT target_point = {0, 0};
+      monitor = MonitorFromPoint(target_point, MONITOR_DEFAULTTOPRIMARY);
+    }
```

Comment:
I don't know that we need this functionality, but I guess it's better for API consistency.